census/census_pandas_ibis.py

```diff
@@ -444,9 +437,8 @@ def run_benchmark(parameters):
 
         if parameters["validation"] and parameters["import_mode"] != "pandas":
             print(
-                "WARNING: validation can not be performed, it works only for 'pandas' import mode, '{}' passed".format(
-                    parameters["import_mode"]
-                )
+                f"WARNING: validation can not be performed, it works only for 'pandas' \
+                    import mode, [{parameters['import_mode']}] passed"
```

Comment:
```suggestion
                    import mode, '{parameters['import_mode']}' passed"
```
this would look nicer when formatted IMHO